src/member-api.js

```diff
@@ -51,15 +51,15 @@ export class MemberApi extends TypedEmitter {
    * @param {string} deviceId
    *
    * @param {Object} opts
-   * @param {import('./capabilities.js').RoleId} opts.roleId
+   * @param {import('./roles.js').RoleIdAssignableToOthers} opts.roleId
    * @param {string} [opts.roleName]
    * @param {string} [opts.roleDescription]
    * @param {number} [opts.timeout]
    *
    * @returns {Promise<import('./generated/rpc.js').InviteResponse_Decision>}
    */
   async invite(deviceId, { roleId, roleName, roleDescription, timeout }) {
-    if (!DEFAULT_CAPABILITIES[roleId]) {
+    if (!ROLES[roleId]) {
```

Comment:
I think this needs to be stricter now, since we do not want to allow inviting someone as a project creator. This should be the "assignable to others" role list, although we should probably exclude the blocked role from that, but maybe as a to-do because I think some tests invite as blocked, just because I was lazy.

Reply:
Done in f24c85e3d807ce58e3f4b99a148f6ee1c9d345b6.